packages/amplify-codegen/package.json

```diff
@@ -25,7 +25,7 @@
     "@aws-amplify/graphql-docs-generator": "4.2.1",
     "@aws-amplify/graphql-generator": "0.4.6",
     "@aws-amplify/graphql-types-generator": "3.6.0",
-    "@graphql-codegen/core": "2.6.6",
+    "@graphql-codegen/core": "^2.6.6",
```

Comment:
We had multiple versions in our node_modules, this consolidates down to a single version which makes it easier to detect if we upgrade.